packages/react-strict-dom/src/native/modules/createStrictDOMComponent.js

```diff
@@ -672,7 +672,7 @@ export function createStrictDOMComponent<T, P: StrictProps>(
         if (spellCheck != null) {
           nativeProps.spellCheck = spellCheck;
         }
-        if (value != null) {
+        if (value != null && typeof value === 'string') {
```

Comment:
Wondering why we added the string check here.

Does `StrictReactDOMListItemProps`'s `value` property change from number to string for this expression to pass?

Reply:
The `value` prop for RN TextInput is `string` and given the way we mash all the element prop types together to type this function, it required a runtime type check